content/specification.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+permalink: /specification/
+Title: Specification
+class: page-short
+---
+
+# Specification
+
+The Common Workflow Language (CWL) project produces free and open standards for describing command-line tool based workflows. These standards are [implemented](/implementations/) in many popular workflow systems. 
+
+For developers and advanced users, the current [CWL Standards v1.2.0](https://www.commonwl.org/v1.2/) provides authoritative documentation of the execution of CWL documents. The previous versions, [CWL Standards v1.0.2](https://www.commonwl.org/v1.0/) and [CWL Standards v1.1.0](https://www.commonwl.org/v1.1/), are also available.
+
+{:.cwl-red}
+## References
+
+To reference the Common Workflow Language and the CWL project in scholarly work, please use the following citation:
+
+* Michael R. Crusoe, Sanne Abeln, Alexandru Iosup, Peter Amstutz, John Chilton, Nebojša Tijanić, Hervé Ménager, Stian Soiland-Reyes, Carole Goble, The CWL Community (2021):
+Methods Included: Standardizing Computational Reuse and Portability with the Common Workflow Language.
+arXiv 2105.07028 [cs.DC] <https://arxiv.org/abs/2105.07028>
+
+To reference the CWL specification in scholary work, please use the following citation inclusive of the DOI:
+
+* Peter Amstutz, Michael R. Crusoe, Nebojša Tijanić (editors), Brad Chapman, John Chilton, Michael Heuer, Andrey Kartashov, Dan Leehr, Hervé Ménager, Maya Nedeljkovich, Matt Scales, Stian Soiland-Reyes, Luka Stojanovic (2016):
+Common Workflow Language, v1.0.
+Specification, Common Workflow Language working group. <https://w3id.org/cwl/v1.0/> <https://doi.org/10.6084/m9.figshare.3115156.v2>
```

Comment:
Maybe include the ready-to-copy-and-paste examples for bibtex and other citation formats, as GitHub does? https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files